docs/developer-guide/using-libraries/child-processes.md

```diff
@@ -309,4 +311,43 @@ In general, it is recommended to call the functions with `pcall()` until this
 inconsistency is fixed in the future.
 
 
+### Writing data to a process
```

Comment:
Please rephrase this to sound similar to https://lite-xl.com/developer-guide/using-libraries/child-processes/#reading-from-a-child-process.